components/integrations/installed_oauth_app.jsx

```diff
@@ -184,6 +194,22 @@ export default class InstalledOAuthApp extends React.PureComponent {
                     />
                 </button>
             );
+            clientSecret = (
+                <span className='item-details__token'>
+                    <FormattedMarkdownMessage
+                        id='installed_integrations.client_secret'
+                        defaultMessage='Client Secret: **{clientSecret}**'
+                        values={{
+                            clientSecret: this.state.clientSecret,
+                        }}
+                    />
+                    <CopyText
```

Comment:
0/5, but I think it makes sense to allow the copy without having to show the secret (useful for over screen snoopers).

Reply:
imho if you copy the secret with the *** in place the user can feel this is not too secure....